ecosystem/typescript/sdk/src/util.test.ts

```diff
@@ -1,5 +1,5 @@
-export const NODE_URL = process.env.APTOS_NODE_URL || "https://fullnode.devnet.aptoslabs.com/v1";
-export const FAUCET_URL = process.env.APTOS_FAUCET_URL || "https://faucet.devnet.aptoslabs.com";
+export const NODE_URL = process.env.APTOS_NODE_URL;
+export const FAUCET_URL = process.env.APTOS_FAUCET_URL;
```

Comment:
Do we need to remove the defaults right now?

Reply:
is there any reason to keep this as running SDK test with local node?